### PR TITLE
zh-trans: add docs/concepts/overview/working-with-objects/field-selectors.md

### PR DESCRIPTION
**Pull request by SataQiu**
zh-trans: add docs/concepts/overview/working-with-objects/field-selectors.md

### CODE REVIEW

File: content/zh/docs/concepts/overview/working-with-objects/field-selectors.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+$ kubectl get pods --field-selector ""
+```
+{{< /note >}}
+
+## 支持的字段
+<!--
+## Supported fields
+-->
+
+受支持的字段选择器因 Kubernetes 资源类型的不同而不同。
```
Comment (tengqm) [inline content/zh/docs/concepts/overview/working-with-objects/field-selectors.md]:
```suggestion
不同的 Kubernetes 资源类型支持不同的字段选择器。
```

File: content/zh/docs/concepts/overview/working-with-objects/field-selectors.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+$ kubectl get pods --field-selector=status.phase!=Running,spec.restartPolicy=Always
+```
+
+## 多种资源类型
+<!--
+## Multiple resource types
+-->
+
+您能够跨多种资源类型来使用字段选择器。
+下面这个 `kubectl` 命令将筛选出所有不在 `default` 命名空间中的 Statefulsets 和 Services :
```
Comment (tengqm) [inline content/zh/docs/concepts/overview/working-with-objects/field-selectors.md]: StatefulSet

Comment (SataQiu) [reply content/zh/docs/concepts/overview/working-with-objects/field-selectors.md]: 非常感谢您的指正！

**Pull request merged**